src/main/kotlin/Node.kt

```diff
@@ -30,6 +30,8 @@ data class Destination(override val host: String, override val port: Int) : Addr
 
 typealias Timestamp = Int //TODO Make this a Comparable, so when we change it to a 'Date' type for the real world, we do not need to change the usages
 
+// TODO: Make a map <Destination, List<ReceivedMessage>> for the received
```

Comment:
Not sure we need this

Reply:
Not anymore, I added to not fall into the cracks. Thanks for checking